### DISCUSSION TITLE
How to verify that my account is a teacher's account and not only a student account?

### DISCUSSION THREAD

**Discussion by zewais** [community/community#138125] [GitHub Education]
### Hi everyone,

Hello,
I applied to GitHub Education as a teacher, and my application got approved, but I only have access to student benefits and not teacher benefits. I need help accessing the teacher's toolbox. I managed to upgrade my organization to GitHub Team instead of the free tier, but I'm confused because the video tutorials say you can access the teacher's toolbox, but I cannot reach it. Advice is appreciated.
Ziad.

**Comment by Shriharsh-Deshmukh** [comment]
Hi Ziad,

To resolve the issue with accessing the GitHub Teacher Toolbox:

1. **Verify Educator Status:** Ensure your GitHub account is flagged as a teacher. Check this under your account settings at [GitHub Education](https://education.github.com/benefits).

2. **Direct Access Attempt:** Try accessing the Teacher Toolbox via [this link](https://education.github.com/toolbox). If it redirects or shows student benefits only, clear your cache or try a different browser.

3. **Validate Organization Upgrade:** Confirm that your organization is correctly upgraded to GitHub Team via Education benefits. Check your organization's billing settings for any discrepancies.

4. **API Check:** You can use the GitHub API to verify your education status by querying your user details. If it shows "student" instead of "teacher," that might explain the issue.

5. **Support Escalation:** If none of the above resolves the problem, escalate directly to GitHub Education Support via their [contact form](https://support.github.com/contact/education) with your approval email and detailed screenshots.

This should get you full access to the teacher’s toolbox.
